What's new in Azure CLI | Ignite 2023 (#4003)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/whats-new-overview.md

---
# What's new in the Azure CLI

This page highlights new features, articles and learning paths for the Azure CLI.

## Azure Copilot for Azure CLI

> [Microsoft Copilot for Azure](https://techcommunity.microsoft.com/t5/azure-infrastructure-blog/simplify-it-management-with-microsoft-copilot-for-azure-save/ba-p/3981106) (preview) is published! Copilot for Azure is an AI companion that helps you design, operate, optimize, and troubleshoot your cloud infrastructure and services. This benefits Azure CLI users because the knowledge of Azure CLI is built into Copilot.

Reviewer: Please use the link to the official doc: https://aka.ms/MicrosoftCopilotforAzureDocs
For the description, can you also align with the description from the Copilot for Azure docs page?

Ask Copilot for Azure a question about managing Azure resources using Azure CLI! For example:

> - How do I create a virtual machine with Azure CLI?

Reviewer: ```suggestion
- I want to create a virtual machine usign Azure CLI
```

- How do I update service principal credentials using Azure CLI?

> - Where can I find an `az rest` example?

Reviewer: ```suggestion
- I want to create a web app using Azure CLI
```
